src/components/PopularSkillsModule.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { get, flow, pick, map } from 'lodash/fp'
+import { sendGraphqlQuery } from '../actions/graphql'
+import { connectedListProps } from '../util/caching'
+import Avatar from './Avatar'
+import A from './A'
+import { peopleUrl } from '../routes'
+
+const { func, object, array } = React.PropTypes
+
+const fetchPopularSkills = slug =>
+  sendGraphqlQuery(`query ($slug: String) {
```

Comment:
This reads more clearly to me because the `addDataToStore` key is associated with the relevant contents rather than the whole function. We can remove the repetitive use of `data` by post-processing the response from the API with `meta.then`:

```js
return {
  type: SEND_GRAPHQL_QUERY,
  payload: {...},
  meta: {
    subject,
    id,
    then: resp => resp.data
  }
}

Which reminds me that we have to handle GraphQL errors properly. But I can do that later, once this is merged.
```